test/scripts/transforms/test_osiris_transform.py

```diff
@@ -245,7 +245,7 @@ def generate_spec_calibration_workspace():
 def test_osiris_transform_spectroscopy():
     """Test spectroscopy transform"""
     reduction = Mock()
-    reduction.reduction_inputs = {
+    reduction.inputs = {
```

Comment:
These should probably be called job now?

Reply:
Another good spot